dbus/mako.c

```diff
@@ -45,8 +45,18 @@ static int handle_invoke_action(sd_bus_message *msg, void *data,
 		sd_bus_error *ret_error) {
 	struct mako_state *state = data;
 
+	uint32_t id = 0;
+	int ret = sd_bus_message_read(msg, "u", &id);
```

Comment:
You can merge both `sd_bus_message_read` calls, with something among the lines of:

```
int ret = sd_bus_message_read(msg, "us", &id, &action_key);
```